tests/integration/records/test_api_literature.py

```diff
@@ -492,7 +492,7 @@ def test_dump_for_es(base_app, db):
     assert "control_number" in dump
     assert record["control_number"] == dump["control_number"]
     assert "id" in dump
-    assert str(record.id) == dump["id"]
+    assert record["control_number"] == dump["id"]
```

Comment:
We should add a regression test for indexing, we can create a record and make sure is accessible in the search